docs/installation/ubuntulinux.md

```diff
@@ -190,8 +190,11 @@ When users run Docker, they may see these messages when working with an image:
     WARNING: Your kernel does not support cgroup swap limit. WARNING: Your
     kernel does not support swap limit capabilities. Limitation discarded.
 
-To prevent these messages, enable memory and swap accounting on your system. To
-enable these on system using GNU GRUB (GNU GRand Unified Bootloader), do the
+To prevent these messages, enable memory and swap accounting on your
```

Comment:
Fine by me. Do you want me to amend my commit or will you just commit your proposition?